src/example/p4p5/tests/test_setup.py

```diff
@@ -14,19 +20,47 @@ class TestSetup(unittest.TestCase):
     def setUp(self):
         """Custom shared utility setup for tests."""
         self.portal = self.layer['portal']
-        self.installer = api.portal.get_tool('portal_quickinstaller')
+        self.qi = api.portal.get_tool('portal_quickinstaller')
 
     def test_product_installed(self):
         """Test if example.p4p5 is installed with portal_quickinstaller."""
-        self.assertTrue(self.installer.isProductInstalled('example.p4p5'))
-
-    def test_uninstall(self):
-        """Test if example.p4p5 is cleanly uninstalled."""
-        self.installer.uninstallProducts(['example.p4p5'])
-        self.assertFalse(self.installer.isProductInstalled('example.p4p5'))
+        self.assertTrue(self.qi.isProductInstalled(PROJECTNAME))
 
     def test_browserlayer(self):
         """Test that IExampleP4P5Layer is registered."""
         from example.p4p5.interfaces import IExampleP4P5Layer
         from plone.browserlayer import utils
         self.assertIn(IExampleP4P5Layer, utils.registered_layers())
+
+    def test_css_resources(self):
+        css_resource_ids = self.portal.portal_css.getResourceIds()
```

Comment:
It will fail on Plone 5, because CSS are not in portal_css anymore.
I think it would be more accurate to test if the script is in the HTML output of a given page.